src/modules.js

```diff
@@ -141,17 +157,16 @@ if (__analytics.amplitude.isEnabled) {
     });
 
     return {
+      name: 'amplitude',
       trackView(viewName) {
         window.amplitude.getInstance().logEvent('Page Viewed', {page: viewName});
       },
       trackEvent(eventName, properties) {
-        const cleanProperties = {};
-        Object.keys(properties).forEach((propName) => {
```

Comment:
```
JSON.parse(JSON.stringify(obj))
```
remove `undefined` properties but it keep `null`

Reply:
Q: Previous both `undefined` and `null` properties where not sent to the analytic service. Maybe we can retain that logic?

Reply:
ok, if it's important for us then I will be retain it